
The dyn_start is _dyn_start ...

  - Rene Rebe <rene@exactcode.de>

--- ./sparc64/start.S.vanilla	2005-05-07 18:44:38.000000000 +0000
+++ ./sparc64/start.S	2005-05-07 18:44:44.000000000 +0000
@@ -30,7 +30,7 @@
 
 /* Let libc do the rest of the initialization, and call main.  */
 #ifdef WANT_DYNAMIC
-	call	dyn_start
+	call	_dyn_start
 #else
 #ifdef WANT_STACKGAP
 	call	stackgap
